docs/warnings.rst

```diff
@@ -27,7 +27,7 @@ script using the `warnings.filterwarnings` function as follows::
      >>> import warnings
      >>> from astropy.io import fits
      >>> warnings.filterwarnings('ignore', category=UserWarning, append=True)
-     >>> fits.writeto(filename, data, clobber=True)
```

Comment:
@astrofrog, that's a good point and maybe I misunderstood. I thought the point of the example was that overwriting a file, even when using the newer `overwrite` caused a warning of some sort. But maybe that's mistaken, in which case the `overwrite` argument should just be removed from the examples altogether.